content-accelerator/latest/develop/extension-content-accelerator.md

```diff
@@ -2,20 +2,34 @@
 title: Extension Content Accelerator (Custom Amp)
 ---
 
-The Extension Content Accelerator is a generic content accelerator for adding custom configurations and code to an existing accelerator.
+One of the common ways customers and implementation teams extend the out-of-the-box ACA functionality is by using what we refer to as a "Custom Amp". This is essentially a way to use the Alfresco SDK to generate an Alfresco Amp file that can be deployed to Alfresco Content Services for the following functionalities: 
 
-## How to add the Extension Content Accelerator onto existing Accelerators
+* Deploy custom content models 
+* Create custom rest endpoints
+* Add custom Alfresco behaviors 
+* Override existing OpenContent code 
+* Create custom document overlays/watermarks 
 
-There are a few requirements that your custom AMP must meet to work with the existing accelerators.
+This Custom Amp framework can also be thought of as a generic content accelerator for adding custom configurations and code to an existing ACA accelerator.
 
-1. The AMP configuration and code must reside under `alfresco/module/com.alfresco.aca.accelerator.extension` to be properly picked up
+The Offering Management team has generated a sample SDK project (see https://github.com/Alfresco/om-content-accelerator-enterprise-viewer) for reference as a starting point for developing your own Custom Amp. The sample Amp has been developed in a way that it can be used for extension of Claims, Policy and Procedure or Human Resources Employee File Management accelerators.
```

Comment:
There are two suggested changes: 

1. URL format for external links.
2. Replace "Amp" with "AMP" as it's an abbreviation for [Alfresco Module Package](https://docs.alfresco.com/content-services/latest/install/zip/amp/), which is linked at the end of this page.

```suggestion
The Offering Management team has generated a sample SDK project (see [https://github.com/Alfresco/om-content-accelerator-enterprise-viewer](https://github.com/Alfresco/om-content-accelerator-enterprise-viewer){:target="_blank"} for reference as a starting point for developing your own Custom AMP. The sample AMP has been developed in a way that it can be used for the extension of Claims, Policy and Procedure, or Human Resources Employee File Management accelerators.
```